extras/gizmo/mesh-tri-data.js

```diff
@@ -0,0 +1,74 @@
+import {
+    Mesh,
+    Mat4,
+    Quat,
+    Vec3
+} from 'playcanvas';
+
+import { Tri } from './tri.js';
+
+// temporary variables
+const tmpV1 = new Vec3();
+const tmpV2 = new Vec3();
+const tmpV3 = new Vec3();
+
+/**
+ * The class for holding mesh triangle data.
+ */
+class MeshTriData {
+    /**
+     * The local to parent transform matrix.
```

Comment:
what parent? Is this a hierarchy?
It's just a world transform for the triangles.

Reply:
So that matrix is for the transformation of the mesh relative to its parent specifically for an `AxisShape` consisting of more than one primitive mesh.